common/tests/thrift_router_test.cpp

```diff
@@ -23,9 +23,11 @@
 #include <tuple>
```

Comment:
I think most of the changes to existing tests can be undone, right?
Because we now rely on client passing in local_group for routing.
We just need to add a new test for the new use case.